### ISSUE TITLE
Mini player should be able to play videos

### ISSUE THREAD

**Issue by aedalzotto** [ytmdesktop/ytmdesktop#414]
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [x] I verified that there is no open issue for the same subject.

When using miniplayer with videos, it only shows the thumbnails.

Miniplayer should play the video instead, just like the behavior of using Youtube Music web page with Floating Player extension.

**Comment by jippylong12** [comment]
This looks like it would be a great enhancement. The mini player feature itself is still on a feature branch (`v2-feature/miniplayer`) and hasn't been merged into the main codebase yet. It's possible it may have been forgotten about. It might be worth revisiting the mini player feature so that this could be addressed once it lands.
